middlewares/hacker.middleware.js

```diff
@@ -356,7 +355,7 @@ function checkStatus(statuses) {
 
         let hacker = await Services.Hacker.findById(req.params.id);
 
-        if (!!hacker) {
```

Comment:
There was an original reason why we did '!!' right?

Reply:
The original reason was to convert to a Boolean, but eslint considers it unnecessary.